src/contact/components/DisplayInfo.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+class DisplayInfo extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+  
+  componentDidMount() {
+    if(this.props.info === null){
+      this.props.actions.displayInfo(false);
+    }
+  }
+
+  onEdit(){
+    this.props.history.push('/edit');
+  }
+
+  render() {
+    //fallback UI
+    if (this.props.hasError || this.props.info === null) {
+      return (
+        <div>
+          <h1>Oops, something went wrong :(</h1>
+          <p>Error in Loading</p>
+        </div>
+      );
+    }
+    if(this.props.info){
+      const {title,subtitle,firstname,lastname,primaryemail,primaryphno,otheremails,otherphnos} = this.props.info;
+      return (
+          <div className = "contact-info">
+          
+          <h1>{title}</h1>
+          <div className = "sub-info">
+            <h2>{subtitle}</h2> 
```

Comment:
This is poorly structured markup from semantic perspective.

Reply:
This can be made in ul li structure.

Reply:
structure modified